src/sentry/static/sentry/app/actionCreators/organization.jsx

```diff
@@ -30,6 +32,22 @@ export async function fetchOrganizationDetails(api, slug, detailed, silent) {
       return;
     }
 
+    // create a request for all teams and all projects if in lightweight org
+    if (!detailed) {
+      // create a new client so when this context unmounts the requests continue
+      const persistedAPI = new Client();
```

Comment:
* I think we try to use camel case even for abbreviations so there's less ambiguity
* Maybe `uncancelable` instead of `persisted`? Persisted has other connotations that might confuse people.